src/pages/Register/Register.jsx

```diff
@@ -0,0 +1,91 @@
+import axios from "axios";
+import React, { useState } from "react";
+import { useNavigate } from "react-router-dom";
+import { Navbar } from "../../components";
+import { useAuth } from "../../context";
+
+const Register = () => {
+  const [firstName, setFirstName] = useState("Humza");
+  const [email, setEmail] = useState("foody@gmail.com");
+  const [password, setPassword] = useState("123456");
```

Comment:
same single state as an object

Reply:
Good suggestion will refactor it in coming PR.